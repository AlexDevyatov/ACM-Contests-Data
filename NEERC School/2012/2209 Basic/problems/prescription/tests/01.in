house_g_0101_first_january_angioplasty
