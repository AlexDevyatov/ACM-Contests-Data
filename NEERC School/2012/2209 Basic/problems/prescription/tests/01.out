hose_g_0101_first_janary_angioplasty
